docs/PROCEDURES.md

```diff
@@ -0,0 +1,69 @@
+# Additional procedures for code maintenance 
+
+## Managing pinned dependencies
+
+The OpenSSF, via the [scorecard](https://securityscorecards.dev/) project recommends that projects pin any
+dependencies they use:
+
+* to ensure reproducibility
+* to reduce the change for rogue dependency updates to compromise software
```

Comment:
Don't understand the term "reduce the change": Do you mean "reduce the risk"?

Reply:
I did mean that - thanks for spotting.